src/Components/Shifting/ShiftDetailsUpdate.tsx

```diff
@@ -245,11 +245,11 @@ export const ShiftDetailsUpdate = (props: patientShiftProps) => {
   }
 
   return (
-    <div className="px-2 pb-2">
-      <PageTitle
-        title={t("update_shift_request")}
-        backUrl={`/shifting/${props.id}`}
-      />
+    <Page
+      title={t("update_shift_request")}
+      backUrl={`/shifting/${props.id}`}
+      className="px-2 pb-2"
```

Comment:
Avoid using custom padding when using `Page` component unless necessary. Newer components allows us to have consistent designs across the platform out of the box.
```suggestion
```